src/app/(proper_react)/redesign/(public)/HeresHowWeHelp.tsx

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { signIn } from "next-auth/react";
+import { Button } from "../../../components/client/Button";
+import { useL10n } from "../../../hooks/l10n";
+import styles from "./LandingView.module.scss";
+import { CheckIcon } from "../../../components/server/Icons";
+import { useTelemetry } from "../../../hooks/useTelemetry";
+
+export const HeresHowWeHelp = () => {
+  const l10n = useL10n();
+  const record = useTelemetry();
+
+  return (
+    <section className={styles.heresHowWeHelpWrapper}>
```

Comment:
Like in the other PR, I think the `<section>`s should be `<div>`s, since it's wrapped in a `<main>` (which acts as a section).